#! /usr/bin/env python
# Create a cfg file for getting control parameters for dmoc optimal controller

PACKAGE = 'rqt_quadcoptergui'

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

#       Name               , Type     , Reconfiguration level , Description                      , Default , Min   , Max
gen.add("vx"              , double_t , 0                     , "Body Velocity x"                 , 0.0     , 0       , 5	)
gen.add("vy"              , double_t , 0                     , "Body Velocity y"                 , 0.0     , 0     , 5	)
gen.add("vz"              , double_t , 0                     , "Body Velocity z"                 , 0.0     , 0     , 5	)
gen.add("update_vel"      , bool_t   , 0x0002                , "Update Goal "                   , False)
exit(gen.generate(PACKAGE , "Config" , "QuadcopterInterface"))




